<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicoCWidget</class>
 <widget class="QWidget" name="PicoCWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PicoC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../coreplugin/core.qrc">
    <normaloff>:/core/gcs_logo_64</normaloff>:/core/gcs_logo_64</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="tbNewFile">
       <property name="toolTip">
        <string>New file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/new archive.png</normaloff>:/picoc/images/new archive.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbLoadFromFile">
       <property name="toolTip">
        <string>Read from file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/unarchive.png</normaloff>:/picoc/images/unarchive.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbSaveToFile">
       <property name="toolTip">
        <string>Save to file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/Ekisho Deep Ocean HD1.png</normaloff>:/picoc/images/Ekisho Deep Ocean HD1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbFetchFromUAV">
       <property name="toolTip">
        <string>Fetch from UAV</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/down_alt.png</normaloff>:/picoc/images/down_alt.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbSendToUAV">
       <property name="toolTip">
        <string>Send to UAV</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/up_alt.png</normaloff>:/picoc/images/up_alt.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbStartScript">
       <property name="toolTip">
        <string>Start Script on UAV</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/execute.png</normaloff>:/picoc/images/execute.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbFileID">
       <property name="toolTip">
        <string>FileID</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbFetchFromUAVROM">
       <property name="toolTip">
        <string>Read from UAV file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/down_rom.png</normaloff>:/picoc/images/down_rom.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbSendToUAVROM">
       <property name="toolTip">
        <string>Save to UAV file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/up_rom.png</normaloff>:/picoc/images/up_rom.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbEraseFromUAVROM">
       <property name="toolTip">
        <string>Delete UAV file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/trash.png</normaloff>:/picoc/images/trash.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="ptEditor">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="tabStopWidth">
      <number>35</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QSpinBox" name="sbExitValue">
       <property name="toolTip">
        <string>Exit Value</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>-32768</number>
       </property>
       <property name="maximum">
        <number>32768</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbTestValue">
       <property name="toolTip">
        <string>Test Value</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-32768</number>
       </property>
       <property name="maximum">
        <number>32767</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTestValueSend">
       <property name="toolTip">
        <string>Send Test Value to UAV</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="picoc.qrc">
         <normaloff>:/picoc/images/up_edit.png</normaloff>:/picoc/images/up_edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>19</width>
         <height>19</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="pbPercent">
       <property name="toolTip">
        <string>Progress</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../coreplugin/core.qrc"/>
  <include location="../coreplugin/core.qrc"/>
  <include location="picoc.qrc"/>
 </resources>
 <connections/>
</ui>
